dashboards/memcached/memcached-prometheus.json

```diff
@@ -1,5 +1,5 @@
 {
-  "category": "CUSTOM",
+  "displayName": "Memcached Prometheus Overview - KPI/LAYOUT UPDATES",
```

Comment:
Missed this earlier -- drop " - KPI/LAYOUT UPDATES" please

Reply:
Sorry about that, should be good to go.